composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": "^7.2",
         "dingo/api": "^2.3",
```

Comment:
If the goal is to upgrade to Laravel 6, i suggest to change laravel/framework to "^6.0", but also dingo/api to "^2.4", laravel/passport to "^7.5", spatie/laravel-permission to "^3.0"

Reply:
Also, beyondcode/laravel-dump-server is deprecated for facade/ignition "^1.4"
Finally, i also suggest phpunit/phpunit  to "^8.0"

Reply:
I agree, At first i thought it was just a composer update, but if this is for the issue https://github.com/joselfonseca/laravel-api/issues/47 then the actual update is to Laravel 6 which means updating all the packages and updating the actual app directory and config files.